Dockerfile

```diff
@@ -16,6 +16,7 @@ ADD server.py .
 
 # Add your model weight files 
 # (in this case we have a python script)
+ARG HF_AUTH_TOKEN
```

Comment:
suggest adding an ENV line below, which explicitly sets HF_AUTH_TOKEN in the dockerfile. We can comment it out and say "this is optional if you don't want to take the more secure build arg route"